api/config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   env: env,
   port: process.env.PORT || '4000',
   queryPath: '/',
+  frontendURL: "https://app.lunie.io",
   transactionPath: '/transaction',
```

Comment:
lint please

Reply:
linter doesn't complain 🤷‍♂️

Reply:
:shrug: we never solved the different linter opinions then after all?